src/app/globals.css

```diff
@@ -37,7 +37,7 @@
     --success-foreground: 129 65% 53%;
 
     --destructive: 353 100% 51%;
-    --destructive-foreground: 210 40% 98%;
+    --destructive-foreground: 352, 100%, 36%;
```

Comment:
this color did not match the figma token 💀

Reply:
hahahaha F